### ISSUE TITLE
Add facets for ENVO

### ISSUE THREAD

**Issue by wdduncan** [microbiomedata/pilot#12]
As terms /data are mapped to the [environmental ontology](https://github.com/EnvironmentOntology/envo) (ENVO ), we will want to be able search facets based on ENVO terms.

**Comment by kfagnan** [comment]
Agreed. This is important.

**Comment by cmungall** [comment]
Obviously this needs to be in the metadata.. I suggest linking to a reciprocal ticket in the md repo to track the dependency.

KitWare folks: you will need to load EnvO separately. LMK if you need advice here, we have a lot of experience of how ontologies are used (and misused..) in various applications

**Comment by jeffbaumes** [comment]
@cmungall yes, a starting point on how you would approach getting the ENVO data linked in would be appreciated.
